src/components/ContactCard/ContactForm/ContactFormField.jsx

```diff
@@ -8,74 +8,32 @@ import Icon from 'cozy-ui/transpiled/react/Icon'
 import palette from 'cozy-ui/transpiled/react/palette'
 import { Media, Img, Bd } from 'cozy-ui/transpiled/react/Media'
 import PlusIcon from 'cozy-ui/transpiled/react/Icons/Plus'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 
-class ContactFormField extends React.Component {
-  constructor(props) {
-    super(props)
-    this.fieldID = uniqueId('field_')
-  }
+const ContactFormField = ({ name, icon, isArray, renderInput }) => {
+  const { t } = useI18n()
 
-  render() {
-    const { name, icon, isArray, renderInput, t } = this.props
-    return (
-      <Media align="top" className="contact-form-field">
-        <Img>
-          {icon ? (
-            <Icon
-              icon={icon}
-              color={palette['coolGrey']}
-              className="contact-form-field__icon u-mr-2"
-            />
-          ) : (
-            <div className="u-w-1 u-mr-2" />
-          )}
-        </Img>
-        <Bd>
-          {isArray ? (
-            <FieldArray name={name}>
-              {({ fields }) => (
-                <div className="u-mt-1 u-mb-half">
-                  {fields.map((nameWithIndex, index) =>
-                    this.renderArrayField(
-                      fields,
-                      index,
-                      nameWithIndex,
-                      name,
-                      renderInput
-                    )
-                  )}
-                  <Button
-                    icon={PlusIcon}
-                    theme="text"
-                    type="button"
-                    onClick={() => this.addField(fields)}
-                    label={t(`addLabel.${name}`)}
-                    className="u-ph-0 u-mh-0"
-                  />
-                </div>
-              )}
-            </FieldArray>
-          ) : (
-            <div className="u-mt-1">{renderInput(name, this.fieldID)}</div>
-          )}
-        </Bd>
-      </Media>
-    )
-  }
+  const fieldID = uniqueId('field_')
 
-  addField = fields => fields.push(undefined)
+  const addField = fields => fields.push(undefined)
 
-  removeField = (fields, index) => {
+  const removeField = (fields, index) => {
     const isLastRemainignField = fields.length === 1
     fields.remove(index)
-    if (isLastRemainignField) this.addField(fields)
+    if (isLastRemainignField) addField(fields)
   }
```

Comment:
`addField` and `removeField` can be move out of the component since there are not dependent of the react life cycle.